app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,26 @@
+<h2>Log in</h2>
```

Comment:
- [ ] Kindly ensure the signin page matches the design given, check the comparison below:

Behance Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/61408860/211101822-c4a79f7a-566c-4956-b5ef-178d97064721.png)| ![image](https://user-images.githubusercontent.com/61408860/211101869-698108c4-d283-46a3-8f9e-898764496d3e.png)